packages/ui/src/proposals/modals/AddNewProposal/helpers.ts

```diff
@@ -402,5 +409,8 @@ export const schemaFactory = (api?: Api) => {
         .required(),
       channelCashoutsEnabled: Yup.boolean(),
     }),
+    updatePalletFrozenStatus: Yup.object().shape({
+      frozen: Yup.boolean().test(equalValueContext('palletFrozenStatus', 'execution')).required('Field is required'),
```

Comment:
Please add a message field to `equalValueContext` so it actually display the error (like on `minContext` and `maxContext`). For example on Set Referral Cut:
![image](https://github.com/Joystream/pioneer/assets/6571453/66cdca17-154f-496f-9230-82ab1f3cc240)

You can see the warning and the box to go further but also there is a message: "Input must be equal or less than 50% for proposal to execute" which explains why the value is wrong.

Here it would be something like:
"The ProjectToken pallet is currently disabled, so presently this proposal would fail due to execution constraints."
Or
"The ProjectToken pallet is currently enabled, so presently this proposal would fail due to execution constraints."